docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+[role="xpack"]
+[[security-getting-started]]
+== Getting started with security
+
+In this tutorial, you learn how to secure a cluster by configuring users and 
+roles for use in {es}, {kib}, {ls}, and {metricbeat}. 
+
+[float]
+[[gs-security-prereqs]]
+=== Before you begin
+
+Install and start {es}, {kib}, {ls}, and {metricbeat} as described in 
+<<get-started-elastic-stack>>. 
+
+Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+[float]
+[[gs-security-license]]
+=== Install a trial license
+
+When you installed the products, they applied Basic licenses with no expiration 
+dates. To view your license in {kib}, go to **Management** and click 
+**License Management**.
+
+[role="screenshot"]
+image::images/management-license.png["License management screenshot in Kibana"]
+
+To secure your cluster, you must obtain a license that includes the {security} 
+features. For more information about Elastic license levels, see 
+https://www.elastic.co/subscriptions.
+
+You can start a 30-day trial to try out all of the Platinum features, including 
+{security}. Click **Start trial**.
+
+IMPORTANT: If your cluster has already activated a trial license for the current 
+major version, you cannot start a new trial. For example, if you have already 
+activated a trial for v6.0, you cannot start a new trial until v7.0.
+
+At the end of the trial period, your license reverts to a Basic license. If you 
+want to keep using the extra features, you can purchase a subscription. 
+
+[float]
+[[gs-security-enabled]]
+=== Enable security in {es}
+
+When you use the trial license, {security} is disabled by default. You must 
+stop {kib}, stop {es}, then add the `xpack.security.enabled` setting to the 
+`config/elasticsearch.yml` file in your {es} directory. For example:
```

Comment:
This is not really specific to this set of docs, but I've noticed we tend to assume that people reading the docs are using the default zip/tar.gz layout with a `config/` directory inside the elasticsearch dir, and not the deb/rpm layout with the config directory in `/etc/elasticsearch` (or the msi install where you get to pick your location, but at least it defaults to being in the elasticsearch directory).

I don't know what we should do to solve that in general, but maybe here we need to have a note for users of the rpm/deb to check the location of their config directory?
- RPM: https://www.elastic.co/guide/en/elasticsearch/reference/6.3/rpm.html#rpm-configuring
- DEB: https://www.elastic.co/guide/en/elasticsearch/reference/6.3/deb.html#deb-configuring

I sometimes get caught out on the forums, where my references to the `config/` directory leave rpm/deb users very confused.

Reply:
At one point we used $CONF_DIR in the x-pack docs because of this

Reply:
Thanks, I hadn't realized it could be in different places. I've added a tip where we talk about the elasticsearch.yml and kibana.yml files.  I've also created https://github.com/elastic/elasticsearch/pull/31635 to clean up path issues in some other security pages.